cpp/utils.h

```diff
@@ -116,4 +116,29 @@ void evaluate_basis_functions(const dolfinx::fem::FunctionSpace& V,
                               const xtl::span<const std::int32_t>& cells,
                               xt::xtensor<double, 3>& basis_values);
 
+/// Compute physical normal
+/// @param[in] n_ref facet normal on reference element
+/// @param[in] K inverse Jacobian
+/// @param[out] n_phys facet normal on physical element
+void compute_normal(const xt::xtensor<double, 1>& n_ref,
+                    const xt::xtensor<double, 2>& K,
+                    xt::xarray<double>& n_phys);
+
+/// Compute jacobians on a given facet
```

Comment:
Could you specify that you compute the Jacobian from physical cell -> reference cell and physical facet -> reference facet. Currently I thought the name indicated that you computed `J_f`